CodeMaid/CodeMaid.csproj

```diff
@@ -46,7 +29,6 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
-    <DeployExtension>True</DeployExtension>
```

Comment:
I'm not sure, but I thought these statements were required for Visual Studio to deploy the extension to the experimental instance on build?